apps/ledger-live-desktop/src/renderer/modals/UpdateFirmwareModal/Cancel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Button, Divider, Flex, IconsLegacy, Text } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import { ErrorBody } from "~/renderer/components/DeviceAction/rendering";
+import Link from "~/renderer/components/Link";
+
+type Props = {
+  onContinue: () => void;
+  onCancel: () => void;
+};
+
+const Cancel = ({ onContinue, onCancel }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <Flex flex={1} flexDirection="column">
+      <Flex flex={1} alignItems="center" justifyContent="center" flexDirection="column" px={100}>
+        <ErrorBody
+          Icon={() => <IconsLegacy.WarningMedium color={"warning.c50"} size={24} />}
+          title={t("manager.firmware.updateInProgress")}
+          description={t("manager.firmware.cancelUpdateWarning")}
+        />
+      </Flex>
+      <Divider />
+      <Flex
+        px={12}
+        alignSelf="stretch"
+        flexDirection="row"
+        justifyContent="space-between"
+        paddingTop={4}
+        paddingBottom={1}
+      >
+        <Flex alignItems="center" justifyContent="center">
+          <Link onClick={onCancel}>
+            <Text fontWeight="600">{t("manager.firmware.cancelUpdate")}</Text>
```

Comment:
We don't use the `font-weight` property often -> is there a `variant` (it's a prop on `Text`) that is closed to what you need ? (you can choose `body`, `h4` etc. it's too have some consistency with the codebase)

Reply:
in that case you should use `import { Link } from "react-ui"` and it's stylised as you want.

Reply:
https://react-ui-storybook.vercel.app/?path=/story/cta-link--default

Reply:
As for regular text (doesn't apply here but just so you know), you should always use the `variant` and `fontWeight` props and never hardcode `fontSize`, `fontFamily` or `fontWeight`.

For instance in the following example, `variant` will be `"bodyLineHeight"` and `fontWeight` will be `"medium"`. You can find all the existing combinations here: https://react-ui-storybook.vercel.app/?path=/story/asorted-typography-text--overview
In case that info is missing on Figma, it's a mistake in the design so ping the designer responsible (or a product manager if you don't know who's the design).
<img width="682" alt="Screenshot 2023-09-14 at 15 35 26" src="https://github.com/LedgerHQ/ledger-live/assets/91890529/0d537561-dad6-4636-b7fd-204cefaad035">